src/pheval/prepare/create_spiked_vcf.py

```diff
@@ -173,20 +246,42 @@ def check_variant_assembly(
 
 
 class VcfSpiker:
-    """Spikes proband variants into template VCF file contents."""
+    """
+    Class for spiking proband variants into template VCF file contents.
+    Attributes:
+        vcf_contents (List[str]): Contents of the template VCF file.
+        proband_causative_variants (List[ProbandCausativeVariant]): List of proband causative variants.
+        vcf_header (VcfHeader): The VCF header information.
+    """
 
     def __init__(
         self,
         vcf_contents: list[str],
         proband_causative_variants: list[ProbandCausativeVariant],
         vcf_header: VcfHeader,
     ):
+        """
+        Initialise the VcfSpiker.
+
+        Args:
+            vcf_contents (List[str]): Contents of the template VCF file.
+            proband_causative_variants (List[ProbandCausativeVariant]): List of proband causative variants.
+            vcf_header (VcfHeader): The VCF header information.
+        """
         self.vcf_contents = vcf_contents
         self.proband_causative_variants = proband_causative_variants
         self.vcf_header = vcf_header
 
     def construct_variant_entry(self, proband_variant_data: ProbandCausativeVariant) -> list[str]:
-        """Constructs variant entries."""
+        """
+        Construct variant entries.
+
+        Args:
+            proband_variant_data (ProbandCausativeVariant): Data for the proband variant.
+
+        Returns:
+            List[str]: Constructed variant entry as a list of strings.
+        """
         genotype_codes = {
             "hemizygous": "0/1",
```

Comment:
I think you mean 'heterozygous' here. hemizygous is for single gene copies in a diploid organism. e.g. for Males in non-PAR X. See also https://bioinformatics.stackexchange.com/questions/14356/how-is-the-gt-field-in-a-vcf-file-defined

Reply:
No, I mean hemizygous. There are some phenopackets in the LIRICAL corpus that are hemizygous. I had checked with you at the time if this is the correct genotype code and you approved

Reply:
Ohhh! I see - it's a lookup for converting things. Sorry, I didn't check the context.